sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -80,7 +81,16 @@ export class StreamingReceiver extends MessageReceiver {
     if (!options) options = {};
     if (options.autoComplete == null) options.autoComplete = true;
     const sReceiver = new StreamingReceiver(context, options);
-    await sReceiver._init();
+
+    const config: RetryConfig<void> = {
+      operation: async () => {
+        await sReceiver._init();
+      },
```

Comment:
Nit: could be simplified to
```suggestion
      operation: () => {
        return sReceiver._init();
      },
```